src/d/actor/d_a_obj_table.cpp

```diff
@@ -4,78 +4,185 @@
 //
 
 #include "d/actor/d_a_obj_table.h"
+#include "d/d_a_obj.h"
+#include "d/d_com_inf_game.h"
 #include "d/d_procname.h"
+#include "f_op/f_op_actor_mng.h"
+
+Mtx daObjTable::Act_c::M_tmp_mtx;
+const char daObjTable::Act_c::M_arcname[6] = "Table";
 
 /* 00000078-000001B8       .text CreateHeap__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::CreateHeap() {
-    /* Nonmatching */
+    u8 prm;
+    J3DModelData *model_data;
+    J3DModel *model;
+
+
+    prm = daObj::PrmAbstract(this, PRM_LINK_ID_W, PRM_LINK_ID_S);
+    if (!prm) {
+        model_data = (J3DModelData*) dRes_control_c::getRes(M_arcname, 4, g_dComIfG_gameInfo.mResControl.mObjectInfo, 0x40);
```

Comment:
Make sure you're checking the debug map for any inlines.
```suggestion
        model_data = (J3DModelData*) dComIfG_getObjectRes(M_arcname, TABLE_BDL_YTBLE);
```
Also, there's an enum in res/res_table.h you can include to make these file indexes more readable.